components/layout_2020/flow/construct.rs

```diff
@@ -143,7 +144,7 @@ pub(crate) struct BlockContainerBuilder<'dom, 'style, Node> {
 
     /// The style of the anonymous block boxes pushed to the list of block-level
     /// boxes, if any (see `end_ongoing_inline_formatting_context`).
```

Comment:
nit: This field now stores more than just the style of the anonymous boxes.

Reply:
Oh, I've fixed this documentation.